lib/ex_plasma/transaction.ex

```diff
@@ -242,4 +245,16 @@ defmodule ExPlasma.Transaction do
     sigs = Enum.map(keys, fn key -> ExPlasma.Encoding.signature_digest(eip712_hash, key) end)
     %{transaction | sigs: sigs}
   end
+
+  # Builds list of utxos and propogates error tuples up the stack.
+  defp build_utxos([]), do: {:ok, []}
+
+  defp build_utxos(utxos) do
+    Enum.reduce_while(utxos, [], fn utxo, acc ->
+      case Utxo.new(utxo) do
+        {:ok, utxo} -> {:cont, {:ok, acc ++ [utxo]}}
```

Comment:
> Also, I'll jump in and add a separate comment to name the "input" utxo differently from the "output" utxo after new

Sorry, could you give an example by what you mean exactly on the naming? 🤔

Reply:
for example:

```
<snip>
  defp build_utxos(rlp_utxos) do
    Enum.reduce_while(rlp_utxos, [], fn rlp_utxo, acc ->
      case Utxo.new(rlp_utxo) do
        {:ok, utxo} ->
<snip>
```

or anything that suits better, so that `utxo` identifier isn't reused